openfecwebapp/templates/partials/search-results-candidate.html

```diff
@@ -8,9 +8,9 @@ <h3 class="result__title">
     </h3>
     <span class="result__years t-data">
       {% if candidate.election_years[0] != candidate.election_years[-1] %}
-        Election years: {{ min(candidate.election_years) }}–{{ max(candidate.election_years) }}
+        Years active: {{ min(candidate.election_years) }}–{{ max(candidate.election_years) }}
```

Comment:
Yea, we changed the behavior in the last few weeks so that it really does use election year to filter on. I did a few tests to confirm and it seems to be filtering by the election year. For example, if you pick a non-presidential cycle and presidential filter, no candidates come up, which would be the desired behavior.